velox/type/Filter.h

```diff
@@ -312,6 +312,10 @@ class AlwaysFalse final : public Filter {
     return false;
   }
 
+  bool testNull() const final {
```

Comment:
Shall we add tests for these methods? Thanks!

Reply:
I would say this is too trivial to be tested